packages/deploy/src/bridge/BridgeContracts.ts

```diff
@@ -122,7 +122,7 @@ export default class BridgeContracts extends AbstractBridgeDeploy<config.EvmBrid
     const factory = new contracts.BridgeToken__factory(this.deployer);
     const impl = await factory.deploy(this.overrides);
     await impl.deployTransaction.wait(this.confirmations);
-
+    
```

Comment:
Revert non-substantive changes like this

Reply:
Reverted